dace/codegen/targets/xilinx.py

```diff
@@ -536,6 +576,58 @@ def generate_module(self, sdfg, state, name, subgraph, parameters,
         kernel_args_module = []
         added = set()
 
+        # Check if we are generating an RTL module, in which case only the
+        # accesses to the streams should be handled
+        rtl_module = False
+        for n in subgraph.nodes():
+            if isinstance(
+                    n, dace.nodes.Tasklet
+            ) and n.language == dace.dtypes.Language.SystemVerilog:
+                rtl_module = True
+                break
+        if rtl_module:
+            entry_stream.write(
+                f'// Placeholder for RTL module: HLSLIB_DATAFLOW_FUNCTION({name})'
+            )
+            module_stream.write(
+                f'// Placeholder for RTL module: void {name}() {{ }}\n\n')
+
+            # _1 in names are due to vitis
+            for node in subgraph.source_nodes():
+                if isinstance(sdfg.arrays[node.data], dace.data.Stream):
+                    if node.data not in self._stream_connections:
+                        self._stream_connections[node.data] = [None, None]
+                    for edge in state.out_edges(node):
+                        rtl_name = "{}_{}_{}_{}".format(edge.dst, sdfg.sdfg_id,
+                                                        sdfg.node_id(state),
+                                                        state.node_id(edge.dst))
+                        self._stream_connections[
+                            node.data][1] = '{}_top_1.s_axis_{}'.format(
+                                rtl_name, edge.dst_conn)
+
+            for node in subgraph.sink_nodes():
+                if isinstance(sdfg.arrays[node.data], dace.data.Stream):
+                    if node.data not in self._stream_connections:
+                        self._stream_connections[node.data] = [None, None]
+                    for edge in state.in_edges(node):
+                        rtl_name = "{}_{}_{}_{}".format(edge.src, sdfg.sdfg_id,
+                                                        sdfg.node_id(state),
+                                                        state.node_id(edge.src))
+                        self._stream_connections[
+                            node.data][0] = '{}_top_1.m_axis_{}'.format(
+                                rtl_name, edge.src_conn)
+
+            # Skip RTL modules
+            # Should still generate the node:
```

Comment:
Maybe make a note of WHY we still need to run generation even though we ignore the generated code.

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/0c848a00b81b0a0907122df943f705271ba26ad8